### DISCUSSION TITLE
Discussion thread for issues/troubleshooting related to the CNS*2022 satellite tutorials

### DISCUSSION THREAD

**Discussion by sanjayankur31** [OCNS/SoftwareWG#103] [General]
Please use this discussion topic to post any issues that you may run into related to the [CNS*2022 satellite tutorials](https://ocns.github.io/SoftwareWG/pages/software-wg-satellite-tutorials-at-cns-2022.html).

You can also drop by on our chat channels either on  [Matrix](https://matrix.to/#/#OCNS_SoftwareWG:gitter.im) or [Gitter](https://gitter.im/OCNS/SoftwareWG). They are bridged, so you can use either.

@OCNS/software-wg : please keep an eye on this discussion.

**Comment by brenthuisman** [comment]
Should the Gitter link be https://gitter.im/OCNS/SoftwareWG ?

**Comment by sanjayankur31** [comment]
Yes, i think so. I'll double check and correct it everywhere.

-- 
Thanks,

Ankur

(Sent from mobile device)
